### DISCUSSION TITLE
Promote this AddOn

### DISCUSSION THREAD

**Discussion by FreePDF64** [okaryo/TabTabTab#118] [General]
Hi,

How is it possible to promote this AddOn? Writing of articles in computer news or anything else... ?

For me is it one of the best Tab Manager for Chromium browsers among other things because of the unique selling point of displaying tab groups.  I would be happy if more users can know/get to know this AddOn.

Best regards,
Michael

**Comment by okaryo** [comment]
Hi Michael,

Thank you so much! I'm really glad to hear that you think highly of the AddOn.

If it's possible, writing articles and sharing them in computer news or on other platforms would be fantastic! Even just sharing TabTabTab on SNS would be greatly appreciated.

Personally, I believe that marketing is essential, not just feature development. In between development tasks, I've been trying various approaches, such as writing tech blogs and experimenting with store images and descriptions. However, it's been a bit challenging to see significant results. As a secret goal for this quarter, I aimed to reach 100 users, but it seems I still have a long way to go.

Once again, thank you for your kind words and support. Every bit of promotion helps, and I hope more users can discover and enjoy the AddOn.

**Comment by FreePDF64** [comment]
Proposal: I would also bring out in the Chrome Web Store add-on overview that TabTabTab also shows tab groups. For me a main feature!

**Comment by okaryo** [comment]
Thank you! I'll update the store description in the next release!
